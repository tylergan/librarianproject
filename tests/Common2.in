add member Tyler
add collection books.csv
rent 100000 111111
relinquish all 100000
rent 100000 111111
relinquish all 100000
common 100000
exit