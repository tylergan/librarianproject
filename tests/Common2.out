user: Success.

user: 17 books successfully added.

user: Success.

user: Success.

user: Success.

user: Success.

user: To Kill a Mockingbird (Harper Lee)

user: Ending Library process.